src/strings/en-us.json

```diff
@@ -955,14 +955,13 @@
     "LabelTitle": "Title",
     "LabelTonemappingAlgorithm": "Select the Tone mapping algorithm to use",
     "LabelTonemappingDesat": "Tone mapping desat",
-    "LabelTonemappingDesatHelp": "Apply desaturation for highlights that exceed this level of brightness. The higher the parameter, the more color information will be preserved. This setting helps prevent unnaturally blown-out colors for super-highlights, by (smoothly) turning into white instead. This makes images feel more natural, at the cost of reducing information about out-of-range colors. The recommended and default values are 0 and 0.5.",
+    "LabelTonemappingDesatHelp": "Apply desaturation for highlights that exceed this level of brightness. The recommended value is 0 (disable).",
     "LabelTonemappingParam": "Tone mapping param",
     "LabelTonemappingParamHelp": "Tune the tone mapping algorithm. The recommended and default values are NaN. Generally leave it blank.",
     "LabelTonemappingPeak": "Tone mapping peak",
-    "LabelTonemappingPeakHelp": "Override signal/nominal/reference peak with this value. Useful when the embedded peak information in display metadata is not reliable or when tone mapping from a lower range to a higher range. The recommended and default values are 100 and 0.",
+    "LabelTonemappingPeakHelp": "Override the embedded metadata value for the input signal with this peak value instead. The default value is 100 (1000nit).",
     "LabelTonemappingRange": "Tone mapping range",
     "LabelTonemappingThreshold": "Tone mapping threshold",
-    "LabelTonemappingThresholdHelp": "The tone mapping algorithm parameters is fine-tuned per each scene. And a threshold is used to detect whether the scene has changed or not. If the distance between the current frame average brightness and the current running average exceeds a threshold value, we would re-calculate scene average and peak brightness. The recommended and default values are 0.8 and 0.2.",
```

Comment:
You should also remove the usage of this?

Reply:
This string is already not used anywhere in web